### ISSUE TITLE
Is this discontinued?

### ISSUE THREAD

**Issue by Edelkrebs** [tuhdo/os01#157]
Just wondering.

**Comment by kosmas12** [comment]
I'm also interested to know if this is still getting made

**Comment by cdecompilador** [comment]
me too

**Comment by Grench6** [comment]
more than a year with no activity... :(

**Comment by Grench6** [comment]
Check these issues: #107 #126

**Comment by kosmas12** [comment]
It seems a PR was merged by the author himself just a couple of hours ago. Are we finally going to see this?

**Comment by Grench6** [comment]
Don't think so... the commits were only minor grammar fixes.

**Comment by kosmas12** [comment]
> Don't think so... the commits were only minor grammar fixes.

Indeed, but the author appeared again after a year of inactivity on GitHub (and over a year of inactivity for the book)
